src/application/App.tsx

```diff
@@ -141,44 +162,47 @@ function LeftSidePanelWrapper() {
   );
 }
 
-function RightPanelWrapper() {
-  const [closed, setClosed] = React.useState(false);
-  const panelRef = React.useRef<ImperativePanelHandle>(null);
+type SecondarySideBarPane = 'Rewriter' | 'Chatbot';
+function RightSidePanelWrapper() {
```

Comment:
What do you think of moving the Left and Right side panel wrappers to own files (or both in same file)?

Reply:
Yes, that makes sense, and will make the code more legible